### PR TITLE
[OO] Scenes Integration Tests

### PR DESCRIPTION
**Pull request by lpbeliveau-silabs**
Added test for scenes integration in onoff cluster.

Note:
Some extra steps had to be added due to the integration with the level control cluster, they will be added to the test plan repo once this PR is approved.

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_OO_2_7.yaml [hunk 1]
```diff
@@ -0,0 +1,314 @@
+# Copyright (c) 2021 Project CHIP Authors
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_OO_2_7.yaml:1]:
```suggestion
# Copyright (c) 2024 Project CHIP Authors
```

File: src/app/tests/suites/certification/Test_TC_OO_2_7.yaml [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneList"
+                value: []
+
+    - label: "Step 2a: TH sends Off command to DUT"
+      PICS: OO.S && !LVL.S.C04.Rsp
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_OO_2_7.yaml:126]: suggest removing the OO.S here - it's already a top level PICS.

And on the next step.

File: src/app/tests/suites/certification/Test_TC_OO_2_7.yaml [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+              - name: "SceneID"
+                value: 0x01
+
+    - label: "Step 3: TH sends a AddScene command to DUT with the GroupID field
+          set to G1, the SceneID field set to 0x02, the TransitionTime field set
+          to 1000 (1s) and the ExtensionFieldSets set to: '[{ ClusterID: 0x0006,
+          AttributeValueList: [{ AttributeID: 0x0000, ValueUnsigned8: 0x01 }]}]'
+          "
+      command: "AddScene"
+      ## The default handlers for level control and on/off assume that if both clusters are in the same endpoint,
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_OO_2_7.yaml:172]: Is that specified somewhere? If someone comes with a different interpretation, will they fail cert and is that as expected?

File: src/app/tests/suites/certification/Test_TC_OO_2_7.yaml [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+      ## the levvel control cluster will use the onoff with level command and thus onoff will be turned on.
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_OO_2_7.yaml:173]:
```suggestion
      ## the level control cluster will use the onoff with level command and thus onoff will be turned on.
```

File: src/app/tests/suites/certification/Test_TC_OO_2_7.yaml (hunk 3 above)
Comment (lpbeliveau-silabs) [reply src/app/tests/suites/certification/Test_TC_OO_2_7.yaml:172]: That is how it is implemented in the SDK as off now, so I do not expect a different interpretation. 

That being said it isn't specified so I added a Spec PR here:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10114

We can have this as a followup, or should we leave the PR on hold?

Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_OO_2_7.yaml:172]: I wouldn't put it past matter.js to be ON it with scenes, btw. But the spec clarification works. As long as they both get it, let's not hold this up on that.

**Review by cecille — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

**Pull request merged**